tests/Unit/StatsTest.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use App\Document;
+use App\Stats;
+
+class StatsTest extends TestCase
+{
+    public $statsObj;
+    public $document;
+
+    public function SetUp()
+    {
+        $path = realpath(__DIR__. '/../test-fixture.txt');
+        $this->document = new Document($path);
+        $this->statsObj = new Stats($this->document);
+    }
+
+    public function testStatsCanBeInitialized()
+    {
+        $this->assertNotNull($this->statsObj);
+        $this->assertNotNull($this->statsObj->numberParagraphs);
+        $this->assertNotNull($this->statsObj->shortestParagraph);
+    }
+
+    public function testStatsCanFindADocumentsTotalNumberOfParagraphs()
+    {
+        $this->assertEquals($this->statsObj->numberParagraphs, 9);
```

Comment:
I understand why you are doing this. And I have done it too - recently even in the Advent of Code - but I really hate fixtures.

It would be so cool if instead the stats object (or other similar objects in the system) just took some text:

```
$document = new Document('This is a test. There are two sentences');
$stats = new Stats($document);

$this->assertEquals($stats->numberSentences(), 2);
```